### DISCUSSION TITLE
Add "Zoom" button to Insight Panels

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#8211] [Feature Requests]
Often times I go into edit mode just to drag a panel larger so I can see it zoomed in for a moment. It would be nice to simply add a hover zoom button that opens the panel at like 80% viewport size, and clicking again quickly closes back to normal.

Not sure if we smoke out the background, open in a drawer, or something else. 🤔

**Comment by rijkvanzanten** [comment]
It's also not _zoom_ as "TV mode" _zoom_ which simply uses a scale transform. I believe what you'd ideally want is that it temporarily sets the width and height to a higher value right?

**Reply by benhaynes** [reply]
Correct. We want to treat it as a larger w/h, not the transform scale.

**Comment by Dumpstar56** [comment]
I actually run into this a lot. I would love a more direct zoom function in the file directory, without having to go into edit mode. But what would be even greater would be a zoom function for the image preview when the file is linked in a m2o or m2m relationship in the db. So something like this:
![zoom](https://user-images.githubusercontent.com/94397183/183448890-cc074c18-f2e6-4969-b93c-a2e34473c976.jpg)

**Reply by rijkvanzanten** [reply]
Mind opening a new feature request discussion for this? The original post here is talking about temporary expanding Insight dashboard panels, not files in the file library 🙂

**Reply by Dumpstar56** [reply]
Done, thank you! https://github.com/directus/directus/discussions/14953

**Comment by licitdev** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
